src/components/side_panel/chart/index.ts

```diff
@@ -60,4 +61,8 @@ chartSidePanelComponentRegistry
   .add("scorecard", {
     configuration: ScorecardChartConfigPanel,
     design: ScorecardChartDesignPanel,
+  })
+  .add("waterfall", {
+    configuration: LineBarPieConfigPanel,
```

Comment:
Maybe we can rename the LineBarPieConfigPanel into something else (like `GeneralConfigPanel`) as it's starting to be used for more and more chart